src/App.jsx

```diff
@@ -1,12 +1,69 @@
 import './App.css'
+import {useEffect, useState} from "react";
+import axios from "axios";
+import PokemonCard from "./components/PokemonCard.jsx";
+import Button from "./components/Button.jsx";
 
 function App() {
+    const [endpoint, setEndpoint] = useState(`https://pokeapi.co/api/v2/pokemon/`);
+    const [pokemons, setPokemons] = useState([])
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(false)
 
-  return (
-    <>
-      <h1>Gotta catch em all!</h1>
-    </>
-  )
+    useEffect(() => {
+        const controller = new AbortController();
+
+        async function getData() {
+            setLoading(true);
+            setError(false);
+
+            try {
+                const result = await axios.get(endpoint, {
+                    signal: controller.signal
+                });
+                setPokemons(result.data);
+                setError(false)
```

Comment:
Error heb je al op false gezet op regel 18, het is niet nodig om dat hier nogmaals te doen.